srcpkgs/libdrm/template

```diff
@@ -32,3 +32,11 @@ libdrm-devel_package() {
 		vmove "usr/lib/*.so"
 	}
 }
+
+libdrm-tests_package() {
```

Comment:
maybe let's call it `libdrm-test-progs`, because they're programs to test the sw, not the test suite of libdrm, right?

Reply:
This was based on https://packages.debian.org/sid/arm64/libdrm-tests/filelist, but you are correct that these are programs to test (rather, poke around) at the kernel software and underlying hardware, not tests that validate the contents/implementation of `libdrm` (in an isolated manner).

Can rename this if `test-progs` is desired, that is after all how the `meson` config property is named.